PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -115,9 +115,10 @@
         'keep *_slimmedGenJetsAK8__*',
         'keep *_slimmedGenJetsAK8SoftDropSubJets__*',
         'keep *_genMetTrue_*_*',
-        # RUN
         'keep LHERunInfoProduct_*_*_*',
         'keep GenRunInfoProduct_*_*_*',
+        'keep GenWeightProduct_*_*_*',
+        'keep GenWeightInfoProduct_*_*_*',
```

Comment:
is there a way to avoid `*` in the module? in reco we usually require that the module names are spelled out explicitly

Reply:
Yes, can be

        'keep GenWeightProduct_genWeights_*_*',
        'keep GenWeightInfoProduct_genWeights_*_*',
        'keep GenWeightProduct_lheWeights_*_*',
        'keep GenWeightInfoProduct_lheWeights_*_*',